### PR TITLE
Bugfix/xxxxx more fixes

### PR DESCRIPTION
**Pull request by cmoesgaard**
- [ ] Have you updated the documentation?
- [ ] Have you performed a smoke test of the application?
- [ ] Have you written tests for your changes?
- [ ] Have you updated the release notes?

### CODE REVIEW

File: backend/mora/service/detail_reading.py [hunk 1]
```diff
@@ -647,6 +647,13 @@ def as_values(vs):
         for classid, classobj in c.klasse.get_all(uuid=class_cache)
     })
 
+    def is_empty_lora_object(obj):
+        empty = not (obj.get('relationer') or
```
Comment (danchr) [inline backend/mora/service/detail_reading.py]: Don't use `get()` when you mean `in`? Alternatively:

```python
return not obj.keys() & {'relationer', 'attributter', 'tilstande'}
```

File: backend/mora/service/detail_reading.py [hunk 2]
```diff
@@ -655,6 +662,7 @@ def as_values(vs):
             **address.get_one_address(funcobj),
         }
         for funcid, funcobj in address_functions.items()
+        if not is_empty_lora_object(funcobj)
```
Comment (danchr) [inline backend/mora/service/detail_reading.py]: Perhaps:

```python
if util.is_reg_valid(funcobj)
```

I believe this is consistent with the other reading code and arguably a more thorough check.

File: backend/mora/service/address_handler/phone.py [hunk 3]
```diff
@@ -66,9 +68,8 @@ def get_lora_properties(self):
     def _get_mo_properties(self):
         properties = super()._get_mo_properties()
         if self.visibility:
+            c = lora.Connector()
             properties.update({
-                mapping.VISIBILITY: {
-                    'uuid': self.visibility
-                }
+                mapping.VISIBILITY: facet.get_one_class(c, self.visibility)
```
Comment (danchr) [inline backend/mora/service/address_handler/phone.py]: Should we cache these?

Comment (cmoesgaard) [reply backend/mora/service/address_handler/phone.py]: Caching classes is some of the most low hanging fruit as they bound to almost never change, so we should definitely do that, but I am not really sure how to do it yet.

I'll add a ticket for it, (if there isn't one already) and then we can talk about the design for it at some point.

File: backend/tests/test_address_handlers.py [hunk 4]
```diff
@@ -336,7 +339,11 @@ def test_get_mo_address(self):
         }
 
         # Act
-        actual = address_handler.get_mo_address_and_properties()
+        with patch(
+            'mora.service.address_handler.phone.facet.get_one_class',
```
Comment (danchr) [inline backend/tests/test_address_handlers.py]: Oy vey, that's a long name 😁  Why not do a proper module-level import and just patch `facet.get_one_class`?

**Pull request merged**